src/storage/slotted_page.h

```diff
@@ -92,8 +92,10 @@ struct spage_slot
 extern void spage_boot (THREAD_ENTRY * thread_p);
 extern void spage_finalize (THREAD_ENTRY * thread_p);
 extern void spage_free_saved_spaces (THREAD_ENTRY * thread_p, void *first_save_entry);
-extern int spage_slot_size (void);
-extern int spage_header_size (void);
+
+#define spage_slot_size()   (sizeof(SPAGE_SLOT))
+#define spage_header_size() (sizeof(SPAGE_HEADER))
```

Comment:
Please move them over the function declaration.

Reply:
OK, I will fix it.

Reply:
Btw, `sizeof(SPAGE_SLOT)` is a fixed size. we use capital characters as code conventions.

e.g) RESERVED_SIZE_IN_PAGE in storage_commmon.c
```
...
#define RESERVED_SIZE_IN_PAGE   (sizeof (FILEIO_PAGE_RESERVED) + sizeof (FILEIO_PAGE_WATERMARK))
...
```

So, they should be
```
#define SPAGE_SLOT_SIZE   (sizeof(SPAGE_SLOT))
#define SPAGE_HEADER_SIZE (sizeof(SPAGE_HEADER))
```

Reply:
I'll fix that, thank you.